.github/workflows/check-markdown-links.yml

```diff
@@ -0,0 +1,24 @@
+name: Check Markdown Links
+
+on:
+  push:
+    branches: ["feat/documentation-link-checker"]
```

Comment:
This requires another change later to set it to the correct branch, right?

Reply:
That is correct. I figure that determining _where_ we want this to be enabled would be a conversation for another PR.